src/certificates/certificates-sidebar/sidebar-block/SidebarBlock.test.jsx

```diff
@@ -0,0 +1,30 @@
+import { render } from '@testing-library/react';
+
+import SidebarBlock from '.';
+
+const testProps = {
+  title: 'Test Title',
+  paragraphs: ['Test Paragraph'],
+};
+
+const renderComponent = (props) => render(
+  <SidebarBlock {...props} />,
+);
+
+describe('SidebarBlock', () => {
+  it('renders without crashing', () => {
+    const { getByText } = renderComponent(testProps);
+    expect(getByText('Test Title')).toBeInTheDocument();
+    expect(getByText('Test Paragraph')).toBeInTheDocument();
```

Comment:
```suggestion
    expect(getByText(testProps.paragraphs)).toBeInTheDocument();
```

Reply:
will be 
`expect(getByText(testProps.paragraphs[0])).toBeInTheDocument();`